### PR TITLE
fix getPermalinkUrl() to not include lang=null param

### PR DESCRIPTION
**Pull request by mmoll**
closes GH-712

### CODE REVIEW

File: js/functions.js [hunk 1]
```diff
@@ -145,8 +145,19 @@ function changeLanguage(lang)
 function getPermalinkUrl(style, lang)
 {
 	var url = window.location.origin + window.location.pathname;
+	var langAppend;
 
-	url += '?lang='+((lang) ? lang : params['lang']);
+	url += '?style='+style;
+
+	if(lang != null) {
```
Comment (DerDakon) [inline js/functions.js:152]: what about something like this instead:

```js
if (lang === null) { lang = params['lang']; }
if (lang !== null) { url += &lang="+lang; }
```

Comment (mmoll) [reply js/functions.js]: ✅

**Review by Nakaner — CHANGES_REQUESTED** [review]
It is `lang=undefined` instead of `lang=null` now.

**Comment by mmoll** [comment]
Should be fixed now.

**Comment by DerDakon** [comment]
You can use the same indentation style like the surrounding code…

**Review by DerDakon — APPROVED** [review]

**Comment by Nakaner** [comment]
The changes were deployed a few minutes ago.

**Pull request merged**